lib/game.rb

```diff
@@ -0,0 +1,104 @@
+require_relative '../lib/board.rb'
+class Game
+  attr_reader :board, :players, :winner, :draw
+
```

Comment:
- [ ] You do not need these attr_readers here. There are not read outside this file and you already have them as class variables in the initialize method.